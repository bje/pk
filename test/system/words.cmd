../../src/fieldx 1 3 5
