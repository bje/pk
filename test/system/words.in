AAA BBBB CCCCC DDDDDD EEEEEEE FFFFFFFF
aaa bbbb ccccc dddddd eeeeeee ffffffff
