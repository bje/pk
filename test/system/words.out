AAA CCCCC EEEEEEE
aaa ccccc eeeeeee
